src/course_bot/csa.clj

```diff
@@ -24,87 +25,91 @@
     :stop "Бот погиб, мой господин!"
     :unknown-1 "Неизвестное сообщение: %s, а вы точно мой господин?"}}})
 
-(def conf (misc/get-config "../edu-csa-internal"))
+(def conf (delay (misc/get-config "../edu-csa-internal")))
```

Comment:
Why we need this instead the normal variables?

Reply:
because uberjar build failed for me before I converted everything related to configs to `future`

Reply:
Also, what is so wrong with it? As it seemed to me, the main point of the issue you created is to decrease the startup time. I can disable AOT and revert everything back to normal variables, but it will lead to compiling at runtime, which definitely doesn't look like a fast startup.

Reply:
Got it. Don't take into account that static java classes work in different way that dynamic clojure modules.

Here I see two approach to finish this issue:

1. Add database check before actual bot startup, it is required because in other case I can restart bot and don't see misconfiguration immediately after that.
2. Move `conf` initialization inside `-main` with `bot-api`. It simplify csa module and API for bot configuration by removing magic symbols.

Selection is up to you.